addons/rose/addon/styles/rose/components/layout/_page.scss

```diff
@@ -32,6 +32,7 @@
 
   .rose-layout-page-body {
     grid-area: body;
+    width: 100%;
```

Comment:
Is there any concern with me adding an explicit `width: 100%` here?

I needed to do this because some of the elements (such as the code editors and token input field) will grow to the size of input, and it looks like we only have a relative `66%` width on the form but nothing on the parents.

<img width="1840" alt="image" src="https://user-images.githubusercontent.com/5783847/184371237-86fa8a9c-44c3-4147-8cd3-f6f3fb0c4cab.png">
<img width="1840" alt="image" src="https://user-images.githubusercontent.com/5783847/184372623-e32a1a37-9ba4-476a-9916-bab208807748.png">


The alternative is I set a fixed `width` or `max-width` on the `.worker-create-section` elements instead of a percentage but I don't think we normally do that?

I'd love to hear any alternatives that I'm not thinking of.